packages/shared/src/index.ts

```diff
@@ -1,34 +1,28 @@
-/**
- * Errors
+/** The following files are not exported on purpose:
+ * - cookie.ts
+ * - globs.ts
+ *
+ * The following folders are also not exported on purpose:
+ * - react
```

Comment:
Putting this into it's own subpath will ensure that in non-React environments it's not pulled in